### ISSUE TITLE
Please consider supporting WebTransport over http/2

### ISSUE THREAD

**Issue by joshtriplett** [BiagioFesta/wtransport#231]
There's a [draft standard](https://datatracker.ietf.org/doc/draft-ietf-webtrans-http2/) for WebTransport over HTTP/2. This exists primarily for environments that reject HTTP/3 (e.g. restrictive corporate firewalls).

Please consider adding support for this in the future, to allow using WebTransport in more cases.

**Comment by StinkySteak** [comment]
does the WebTransport on HTTP/2 support datagram (UDP)?

**Comment by joshtriplett** [comment]
@StinkySteak No; WebTransport over HTTP/2 only supports reliable transport, so the datagram functionality would need to return an error when attempted. This is why browser implementations of WebTransport indicate to JavaScript code using them whether they support unreliable or only reliable transports.
